spikingjelly/activation_based/triton_kernel/torch2triton/torch2graph.py

```
@@ -67,21 +81,36 @@ def generate_inference_graph(fn: Callable, example_inputs: tuple) -> fx.Graph:
+            local_inputs.append(i.detach())
+        else:
+            local_inputs.append(i)
+
+    # Fast-path: try lightweight FX symbolic trace under no_grad for pure forward tracing.
+    try:
+        with torch.no_grad():
+            traced = fx.symbolic_trace(fn)
+            # Run traced once to ensure graph is materialized for dynamic shapes.
+            _ = traced(*local_inputs)
+            return _optimize_graph(traced)
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)